tools/gfx/persistent-shader-cache.cpp

```diff
@@ -121,13 +161,24 @@ void PersistentShaderCache::addEntry(const slang::Digest& dependencyDigest, cons
         deleteLRUEntry();
     }
 
-    ShaderCacheEntry entry = { dependencyDigest, astDigest };
+    auto lastAccessedTime = duration_cast<duration<double>>(high_resolution_clock::now().time_since_epoch()).count();
```

Comment:
I don't think the duration_cast here is necessary.

Reply:
done